packages/account/src/Components/account-limits/__tests__/account-limits-extra-info.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { isMobile } from '@deriv/shared';
+import AccountLimitsExtraInfo from '../account-limits-extra-info';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => true),
+}));
+
+describe('<AccountLimitsExtraInfo/>', () => {
+    it('should render AccountLimitsExtraInfo component', () => {
+        render(<AccountLimitsExtraInfo message='Lorem Ipsom' />);
```

Comment:
I know it is not related to coding, but why not to use correct placeholder? :)
it is 'Lorem ipsum'
you can check here:
https://www.lipsum.com/
https://en.wikipedia.org/wiki/Lorem_ipsum